pkg/apb/svc_acct.go

```diff
@@ -134,20 +136,27 @@ func (s *ServiceAccountManager) createResourceDir() {
 	}
 }
 
-func (s *ServiceAccountManager) createFile(apbId string) string {
-	rFilePath := filepath.Join(resourceDir(), apbId+".yaml")
+func (s *ServiceAccountManager) createFile(handle string) (string, error) {
+	rFilePath := filePathFromHandle(handle)
 	s.log.Debug("Creating resource file %s", rFilePath)
-	var _, err = os.Stat(rFilePath)
 
-	if os.IsNotExist(err) {
-		var file, err = os.Create(rFilePath)
+	_, err := os.Stat(rFilePath)
+	if err != nil && os.IsNotExist(err) {
```

Comment:
I will update, but this pattern is entirely inconsistent across the codebase. We should decide if this style is grounds to reject PRs going forwards or not, and stick to that.